VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
+
+Dataset
+
+Questions
+
+Theory
```

Comment:
Critic Scores predict Global players better as time goes on. User Scores predict more participation in gaming as time goes on.